staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go

```diff
@@ -96,6 +96,9 @@ func NewPathRecorderMux(name string) *PathRecorderMux {
 
 // ListedPaths returns the registered handler exposedPaths.
 func (m *PathRecorderMux) ListedPaths() []string {
+	m.lock.Lock()
+	defer m.lock.Unlock()
```

Comment:
Can we release the lock before the sort?

Reply:
yes, sure. thanks for the commit.